Contributing.md

```diff
@@ -0,0 +1,116 @@
+
+# Contributing to RealtimeSanitizer
+
+This document describes some helpful tips and tricks for contributing to RealtimeSanitizer. 
+
+RealtimeSanitizer is a part of LLVM, meaning that when in conflict, the two following guides take precedence over this one:
+
+1. [Contributing to LLVM](https://llvm.org/docs/Contributing.html)
+2. [LLVM Coding Standards](https://llvm.org/docs/CodingStandards.html)
+3. [Getting Started with the LLVM System](https://llvm.org/docs/GettingStarted.html)
+
+We highly recommend reading these two guides, as they have a lot of things not covered in this document. 
+
+The goal of this document is to be a minimal addition to the above, providing specific advice for developing RealtimeSanitizer.
+
+# How to get in touch
+
+For RTSan specific development, you can ask questions and collaborate with other developers on the  [RealtimeSanitizer discord server](https://discord.com/invite/DZqjbmSZzZ).
+
+For more general questions:
+
+* [LLVM Discord server](https://discord.com/invite/DZqjbmSZzZ)
+* [LLVM Discourse forum](https://discourse.llvm.org/)
```

Comment:
When writing this section, part of me wanted us to roll our server just into the LLVM server. I worry that we would discuss things that should be discussed in full LLVM context, but that doesn't seem 100% appropriate, as there are no sanitizer specific channels.

May just be something we keep an eye on if/when/(depending how much) we grow. At some point it may make sense to totally fold in to LLVM world

Reply:
Great thinking - yes - very much something to be aware of. Thanks for the heads up